src/oedisi/types/data_types.py

```diff
@@ -2,7 +2,7 @@
 import datetime
 from enum import Enum
 from pydantic import BaseModel, root_validator
-from typing import List, Optional, Union, Tuple
+from typing import Tuple
```

Comment:
Is this a potential mistake by ruff? I think that we're trying to use List from typing, but ruff is replacing them with standard lists. 

If this is an issue, is there a ruff per-file-ignore that we should add to stop this being modified?

Reply:
https://stackoverflow.com/questions/39458193/using-list-tuple-etc-from-typing-vs-directly-referring-type-as-list-tuple-etc

I think it has to do with the version number of python being used. This changed in python 3.9.

As long as we import annotations from future it should work in python 3.7+